models/seqweaver.py

```diff
@@ -0,0 +1,70 @@
+import torch
```

Comment:
add a little docstring before the imports saying this is the seqweaver architecture, cite which publication it's from (i might have forgotten to do this in the other `models` architecture files!)